Tutorials/webapp/rstudio_server/Readme.md

```diff
@@ -0,0 +1,78 @@
+<!-- dx-header -->
+# RStudio Server (DNAnexus Platform App)
+
+This is an example code of wrapping the RStudio Server software into a DNAnexus platform's web applet.
+
+The main source file is `src/rstudio_server.sh`. So, modify this file to change the behavior of the applet.
+
+## Used technologies
+The app relies on the following technologies:
+##### 1. RStudio Server's docker image 
+We use the image from the Rocker project and its default setup instructions. If you would like to tweak RStudio settings , please refer to https://hub.docker.com/r/rocker/rstudio/ .
+
+##### 2. dxFUSE
+To mount the parent DNAnexus project as a local directory, we used dxFUSE ( https://github.com/dnanexus/dxfuse/ ). The project files will appear in the home directory. 
+
+<!-- /dx-header -->
+
+<!-- Insert a description of your app here -->
+## Procedure of creating the applet
+
+
+##### 1. Pull ready-to-use RStudio docker image from Docker Hub to your local computer by running in terminal:
+We are going to pull the image locally and upload it to the DNAnexus project instead of pulling the image directly from the Docker Hub. Caching the image in the project causes applet to load faster, as well as eliminates possibility that the applet will fail to load if the internet connection between DNAnexus and Docker Hub has problems.
+ 
+```bash
+docker pull rocker/rstudio
+```
+
+
+##### 2. Save it to a file `rstudio.docker.gz`:
+```bash
+docker save -o rstudio.docker.gz rocker/rstudio
+```
+
+
+##### 3. Log in
+Log into the DNAnexus platform and navigate to the project where you'd like to upload the RStudio docker image and build the applet.
+
+
+##### 4. Upload the file to your DNAnexus project e.g. to `/rstudio/rstudio.docker.gz`
+```bash
+mkdir /rstudio
+dx cd /rstudio
+dx upload rstudio.docker.gz
+```
+
+
+##### 5. Build and deploy RStudio web applet
+```bash
+# Navigate to the directory with the source code of this applet
+# If you cloned the whole repository then it will be
+# /dnanexus-example-applets/Tutorials/webapp/rstudio_server
+cd .../rstudio_server
```

Comment:
One less '.' in the statement "cd ../rstudio_server"

This section could be a bit more clearer. How about showing tree of the /rstudio_server directory? So that the user knows what files and which level they need to be at to dx build?

Reply:
Here, I actually meant three dots, not the directory one level up. Their folder path will vary. I improved the text; I hope this is better.